portal/src/components/DoDontExample/DoDontExample.scss

```diff
@@ -35,6 +35,12 @@
         max-width: rem(420px);
     }
 
+    &__content {
+        color: var(--jkl-color);
+        background-color: var(--jkl-background-color);
+        padding: jkl.$spacing-s;
+    }
```

Comment:
Ikke nødvendigvis noe man må gjøre nå i første omgang, men kanskje man burde legge inn et valg for visning her (som i komponenteksemplene)? Ellers blir det vel fort til at de fleste utviklerne bare ser eksemplene i dark mode 😅

Reply:
Lyst vs mørkt tema skal ikke være poenget med disse eksemplene, så tenker det ikke er så farlig. Og i dette tilfellet er det likt uansett 😄